README.md

```diff
@@ -74,11 +84,13 @@ const Add1Worker = require('worker!../workers/Add1Worker'); // webpack's worker-
 const add1Worker = new Add1Worker;
 
 const workerMiddleware = createWorkerMiddleware(add1Worker);
+const namedWorkerMiddleware = createWorkerMiddleware(add1Worker, "add-worker");
```

Comment:
Mind adding a comment here to explain `"add-worker"` is the name?

Reply:
✅